src/main/java/net/mtrop/doom/tools/decohack/DecoHackParser.java

```diff
@@ -2851,12 +2980,14 @@ private void addErrorMessage(String message, Object... args)
 		private Integer duration;
 		private Boolean bright;
 		private DEHActionPointer action;
-		private Integer parameter0;
-		private Integer parameter1;
+		private Integer misc1;
+		private Integer misc2;
+		private LinkedList<Integer> args;
```

Comment:
Now that I think about it, Line 2979 should also be `List`, too. D'oh!

Reply:
There's a couple of places where `pollFirst` is called on `frameList`, and that method doesn't seem to exist on the general List interface; maybe that's why it was left as a LinkedList?

Will update the args one though, since it doesn't do anything of the sort.

Reply:
Ah, that explains it. should be a `Deque`, then. I'll change it later.